language/move-lang/src/expansion/translate.rs

```diff
@@ -598,7 +598,7 @@ fn module_access(
 // TODO Support uses inside functions. AliasMap will become an accumulator
 
 fn sequence(context: &mut Context, loc: Loc, seq: P::Sequence) -> E::Sequence {
-    let (pitems, pfinal_item) = seq;
+    let (pitems, _, pfinal_item) = seq;
     let mut items: VecDeque<E::SequenceItem> = pitems
```

Comment:
If you use the location on line 608 for the new unit value, it should fix a ton of tests

Reply:
OK. I wasn't sure how you planned to use it.